libcsm/sls/api.py

```diff
@@ -69,8 +68,8 @@ def get_xname(self, hostname: str) -> str:
         """
         try:
             components_response = (self.get_management_components_from_sls()).json()
-        except JSONDecodeError as error:
-            raise JSONDecodeError(f'ERROR did not get valid json for management components \
```

Comment:
Shouldn't we catch both JSONDecoeError and ValueError? Aka lets presume we got html back, that will be a ValueError, but lets say we got partial json back that is likely a decode error.

Reply:
`JSONDecodeError` inherits from `ValueError`.

```python
>>> try:
...     raise JSONDecodeError('foo', 'bar', 0)
... except ValueError as error:
...     print('caught error: %s' % error)
...
caught error: foo: line 1 column 1 (char 0)
```